editor/modes/text-editor/index.js

```diff
@@ -13,19 +13,21 @@ function TextEditor( { blocks, onChange } ) {
 	return (
 		<div>
 			<header className="editor-text-editor__formatting">
-				<button className="editor-text-editor__bold">b</button>
-				<button className="editor-text-editor__italic">i</button>
-				<button className="editor-text-editor__link">link</button>
-				<button>b-quote</button>
-				<button className="editor-text-editor__del">del</button>
-				<button>ins</button>
-				<button>img</button>
-				<button>ul</button>
-				<button>ol</button>
-				<button>li</button>
-				<button>code</button>
-				<button>more</button>
-				<button>close tags</button>
+				<div className="editor-text-editor__formatting-inner">
```

Comment:
In my effort to rid this world of superfluous wrapping components, I might suggest the following alternatives to the issue of padding being affected by the left offset:

- Move the entire editor canvas into a fixed position container and assign the headers as top-assigned absolute within. It's probably not great for maintainability to have non-root components consider sidebar and admin bar offsets.
- Use a different unit for the padding: a pixel or `vw` value perhaps